contracts/user-contracts/src/main/java/org/java/mentorship/contracts/user/client/UserFeignClient.java

```diff
@@ -36,4 +37,22 @@ public interface UserFeignClient {
             value = "/verify/{userId}/{token}"
     )
     Boolean verifyUser(@PathVariable("userId") Integer userId, @PathVariable("token") String token);
+
+    @RequestMapping(
+            method = RequestMethod.POST,
+            value = "/verify/{userId}"
+    )
+    Boolean resendVerificationToken(@PathVariable("userId") Integer userId);
+
+    @RequestMapping(
+            method = RequestMethod.POST,
+            value = "/changepassword/{userId}"
+    )
+    Boolean sendPasswordChangeRequest(@PathVariable("userId") Integer userId);
+
+    @RequestMapping(
+            method = RequestMethod.PUT,
+            value = "/changepassword/{userId}"
+    )
+    Boolean changePasswordWithToken(@PathVariable("userId") Integer userId, @RequestBody PasswordChangeRequest passwordChangeRequest);
```

Comment:
In my opinion this should be a PATCH operation since it is as close to REST as it could get

PATCH @ /users/{userId}?password={hashedPassword}
And the request body should contain the new password